content/hi/docs/contribute/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+content_type: concept
+title: Contribute to K8s docs
+linktitle: Contribute
```

Comment:
The bit after the `:` on these lines should be translated to Hindi.

Reply:
thanks, @sftim . The suggestion is now being implemented.